browser/src/control/jsdialog/Widget.SidebarContainers.ts

```diff
@@ -67,6 +67,14 @@ JSDialog.panel = function (
 			{
 				type: 'toolitem',
 				command: expanderData.command,
+				aria: {
+					label: data.children[0].text
+						? _('More options for %name').replace(
+								'%name',
+								data.children[0].text,
```

Comment:
please use `expanderData` instead of `data`, there we already casted children to Array of any. `text` property should be valid then